templates/industrial/index.html

```diff
@@ -0,0 +1,769 @@
+{% extends "industrial/base_industrial.html" %}
+{% block title %}App-centric Industry 4.0{% endblock %}
+{% block meta_description %}The Ubuntu platform enables industry 4.0 digital transformation. Build industrial IoT appliances for the factory of the future with Ubuntu. Bring cloud native technologies to the factory. Download our tools and start innovating today{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1QiU75g49lRKebt82oyLSxR3-PU96Sk87nqauAZ4vvIY/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>App-centric industry 4.0</h1>
+      <p>Workload consolidation and micro clouds for the factory of the future</p>
+      <p><small>Rated ‘Excellent’ for industrial edge cloud platforms in the <a href="/engage/pac-radar">PAC Radar analyst report</a></small></p>
+      <p><a class="js-invoke-modal p-button--positive" href="/internet-of-things/contact-us?product=industry">Get in touch</a></p>
+    </div>
+    <div class="col-4 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/3d44acdd-Robotics_icon.svg",
+          alt="",
+          height="250",
+          width="248",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru">
+  <div class="row">
+    <blockquote class="p-pull-quote">
+      <p class="p-pull-quote__quote">Building our new ctrlX AUTOMATION with Ubuntu Core has enabled us to create a software-driven industrial manufacturing solution.</p>
+      <cite class="p-pull-quote__citation">Hans-Michael Krause, Director Product Management PLC and IoT, Bosch Rexroth.</cite>
+    </blockquote>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+    <div class="row">
+      <div class="col-6">
+        <h2>Empowering industry 4.0 champions</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
```

Comment:
```suggestion
      <div class="col-10 col-start-large-2">
```

Reply:
To do as Spencer asked you can do this @ClementChaumel

Reply:
Thanks!!